base_export_email/ir_actions_view.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="view_server_action_form" model="ir.ui.view">
+            <field name="name">Server Action (base_export_email)</field>
+            <field name="model">ir.actions.server</field>
+            <field name="inherit_id" ref="base.view_server_action_form"/>
+            <field name="arch" type="xml">
+                <field name="model_id" position="attributes">
+                    <attribute name="on_change">onchange_model_id(model_id, context)</attribute>
+                </field>
+                <field name="model_id" position="after">
+                    <field name="model" invisible="1"/>
+                </field>
+                <field name="srcmodel_id" position="attributes">
+                    <attribute name="attrs">{'required': [('state', '!=', 'export_email'),
+                                                          ('state','!=','dummy'),
+                                                          ('state','!=','sms'),
+                                                          ('state','!=','code'),
+                                                          ('state','!=','loop'),
+                                                          ('state','!=','trigger'),
+                                                          ('state','!=','object_copy'),
+                                                          ('state','!=','client_action'),
+                                                          ('state','!=','email'),
+                                                          ('state','!=','sms'),
+                                                          ('state','!=','other')]}</attribute>
```

Comment:
This long list comes from the standard.
I need it just to add "('state', '!=', 'export_email')"

A better solution is maybe to rewrite it with "('state', '=', ...)" but I tried to stay as to the standard.